index.sh

```diff
@@ -11,32 +11,32 @@
 set -e
 
 # Arguments:
-# 1 - folder to search files in
-function buildIndex() {
-    metaInfoFields=('displayName' 'description' 'tags' 'icon' 'globalMemoryLimit')
+# 1 - folder in which to search for meta.yaml files
+if [[ ! $1 ]]; then echo "Error: must specify path to devfiles, eg., $0 /path/to/che-devfile-registry/devfiles"; exit 1; fi
+devfilesDir=$1; devfilesDir=${devfilesDir%/} # trim trailing slash if present
 
-    ## search for all devfiles
-    readarray -d '' arr < <(find "$1" -name 'meta.yaml' -print0)
+metaInfoFields=('displayName' 'description' 'tags' 'icon' 'globalMemoryLimit')
 
+function buildIndex() {
     FIRST_LINE=true
     echo "["
 
-    ## now loop through meta.yaml files
-    for i in "${arr[@]}"
-    do
-        if [ "$FIRST_LINE" = true ] ; then
+    ## loop through meta.yaml files
+    for i in $(ls -1 "${devfilesDir}/"*"/meta.yaml" | sort); do
```

Comment:
find requires findutils RPM to be installed; `ls` does the same job with smaller container.